### DISCUSSION TITLE
Mobile App Grammar suggestions

### DISCUSSION THREAD

**Discussion by Node815** [immich-app/immich#243] [General]
I am testing the latest app version from F-Droid  (App Version 1.11.0 build.37) and noticed some potential verbiage areas which could use some polish. :)

In the Backup page:

Backup Albums:  Change verbiage from:
          
"Albums to be backup" 
To:
"Albums to be backed up"

**Backup section** - Change verbiage from:

"Photos and videos from selected albums that are backup" 
To:
"Photos and videos remaining to be backed up".

**Remainder section** - Change verbiage from:

"Photos and videos that has not been backing up from selected albums" 
 To:
"Remaining photos and albums to back up from selection:"

In the **"Server Storage"**  verbiage change from:
"Asset that were being backup"  
To: 
"Total assets backed up"

**Comment by alextran1502** [comment]
This is wonderful, thank you. I was playing with words in those sections and nothing sounds as good as your suggestions
